modules/cudf/test/cudf-data-frame-tests.ts

```diff
@@ -233,6 +233,72 @@ test('DataFrame.gather (indices)', () => {
   expect([...rb]).toEqual([...expected_b]);
 });
 
+describe('Dataframe.head', () => {
+  const a  = Series.new([0, 1, 2, 3, 4, 5, 6]);
+  const b  = Series.new([0.0, 1.1, 2.2, 3.3, 4.4, 5.5, 6.6]);
+  const c  = Series.new(['foo', null, null, 'bar', null, null, 'foo']);
+  const df = new DataFrame({'a': a, 'b': b, 'c': c});
+
+  test('default index', () => {
```

Comment:
extreme nit: "index" seems like the wrong name/concept here (and throughout), even just "n" would be slightly more clear I think